src/helperFunctions.js

```diff
@@ -295,9 +295,6 @@ class helpers {
       err = 'Please specify an authorization first.'
     }
 
-    if (err) {
-      throw new Error(err)
-    }
```

Comment:
why deleting this?

Reply:
@dipak has better explanation.
IMO, Promise will reject if there is any error i.e. `err` and that `if` block was blocking any GET requests made without auth-header which were needed for unauthenticated tests.

Reply:
it was added here https://github.com/owncloud/owncloud-sdk/pull/637/files#diff-b58c3c88931c8c80492c58e8b2f7f2000efc13c8eb29120bbdcc033f3f815873R298-R300 and is not needed